core/src/epicli/data/common/ansible/playbooks/roles/postgresql/tasks/replication-repmgr-RedHat.yml

```diff
@@ -0,0 +1,167 @@
+---
```

Comment:
Could we reuse the same code for RHEL and Ubuntu since we have dynamic paths now? Most of the code looks the same.